packages/bierzo-wallet/src/routes/transactions/components/DownloadCSV/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { Fab, makeStyles, Theme } from '@material-ui/core';
+import { useTheme } from '@material-ui/styles';
+import Block from 'medulas-react-components/lib/components/Block';
+import CircleImage from 'medulas-react-components/lib/components/Image/CircleImage';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import * as React from 'react';
+
+import { getBorderColor } from '../../../../theme/css';
+import download from '../../assets/download.svg';
+
+export const CSV_PADDING = 20;
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    width: 'auto',
+    justifyContent: 'left',
+    textTransform: 'inherit',
+  },
+  sizeSmall: {
+    height: `${theme.spacing(4)}px`,
+    padding: '0 !important',
+  },
+  text: {
+    padding: `0px ${theme.spacing(1.5)}px`,
+  },
+  secondary: {
+    backgroundColor: 'white',
+    color: theme.palette.text.primary,
+    padding: 0,
+    boxShadow: 'none',
+    border: `1px solid ${getBorderColor(theme)}`,
+    '&:hover': {
+      background: theme.palette.background.default,
+    },
+  },
+  panel: {
```

Comment:
If you will remove this style and implement it directly on `Block` component and if you will remove `text` style as well  then you can remove line `const fabClasses = {` and provide classes object directly to `Fab` component.

Reply:
And what about classes.panel's backgroundColor? This is not accepted as Block's prop.

Reply:
there is `bgcolor` prop available for `Block`

Reply:
You can use `bgcolor` props to set background color fro `Block`